pkgs/applications/misc/nwg-look/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
```

Comment:
```suggestion
{ lib
, stdenv
, fetchgit
, buildGoModule
, cairo
, glib
, go
, gtk3
, libX11
, pkg-config
, substituteAll
, xcur2png
, zlib
```

Reply:
Why, is it just for convention?

Reply:
just something ive been required to do before merges on all my PRs